.gitignore

```diff
@@ -68,3 +68,4 @@ typings.py
 third_party/airflow/docs/build/
 third_party/airflow/build/
 build/ssl/certs/ca-certificates.crt
+localdev/volumes/go/pkg/
```

Comment:
Maybe you should add volumes/go also

Reply:
good idea

Reply:
well i wanted to make sure the volumes/go folder is created when you checkout -- there's a .keep file in there

Reply:
Ah. I thought the .keep was something that shouldn't be committed.  No worries.